db/schema.rb

```diff
@@ -36,8 +46,10 @@
   add_index "seasons", ["start_date"], name: "index_seasons_on_start_date", unique: true, using: :btree
 
   create_table "teams", force: :cascade do |t|
-    t.string "name",     null: false
-    t.string "location", null: false
+    t.string   "name",       null: false
+    t.string   "location",   null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
```

Comment:
I think you should keep them. Timestamps (in particular) are incredibly valuable for tracking down bugs, creating analytics, sorting results, etc, etc. There's almost never a good reason to remove them. If you think there it we should talk about it. `The app doesn't directly use timestamps on the Team model` isn't a good enough reason.